mars/actors/pool/tests/test_gevent_pool.py

```diff
@@ -516,6 +518,8 @@ def testProcessUnpickled(self):
             with self.assertRaises(pickle.PickleError):
                 ref1.send(('create_unpickled',))
 
+    @unittest.skipIf(sys.platform == 'darwin' and 'TRAVIS' in os.environ,
+                     'Skip in Travis CI under MacOS due to open file number limitations')
```

Comment:
Is it possible to modify the limitation of open file number for MacOS?

Reply:
I don't know where is the limitation.

Reply:
@chineking I've already tested that this problem can be simply solved by adding a ulimit -n 1024 in the preparation stage.

Reply:
Great, so we don't have to skip this test, right?

Reply:
Yeah.